manifests/kafka/broker.pp

```diff
@@ -69,7 +72,7 @@
   String $heap_size                                                     = $::confluent::params::kafka_heap_size,
   Boolean $restart_on_logging_change                                    = true,
   Boolean $restart_on_change                                            = true,
-  Variant[String, Array[String]] $zookeeper_connect                     = 'localhost:2181'
+  Variant[String, Array[String]] $zookeeper_connect                     = $::confluent::kafka::broker::zookeeper_connect
```

Comment:
This is a self-referencing variable, probably not a great idea. It would just be `undef`. Did you test this code without having that parameter set? The default is only used when a parameter is not provided.

Reply:
Hmm... Well, that's embarrassing! xD I'm new to Puppet and was even newer when I wrote this. Thank you for your feedback! I'll take a look at this and rework it at a later point. :)

Reply:
For what it's worth, I think that this part of the code was working as designed. The reason I am looking at this is that we have only one of the three zookeeper servers we have listed showing up. (still investigating)

Reply:
OK, so I found the reason for my issue yesterday, not code related at all. All three servers are showing up as desired with the following hieradata:

```
confluent::kafka::broker::zookeeper_connect:
  - zookeeper01:2181
  - zookeeper02:2181
  - zookeeper03:2181
```

Reply:
Hi again, 
Obviously the problem we had must have been solved as of other changes on our part.
I'll close this PR. Thanks for the feedback!